www/js/components/pages/service/index.jsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Copyright (c) 2016, Joyent, Inc.
+ */
+
+'use strict';
+
+var React = require('react');
+var adminui = require('../../../adminui');
+var _ = require('underscore');
+var ServiceModel = require('../../../models/service');
+
+var ErrorAlert = require('../../error-alert');
+var Metadata =  require('../vm/metadata');
+
+var ServicePage = React.createClass({
+    statics: {
+        sidebar: 'services',
+        url: function (props) {
+            return _.str.sprintf('/services/%s', props.uuid);
+        }
+    },
+    getInitialState: function () {
+        return {
+            loading: true
+        };
+    },
+    componentDidMount: function () {
+        var self = this;
+        var service = new ServiceModel({uuid: this.props.uuid});
+        service.fetch().done(function (model) {
+            self.setState({loading: false, service: model});
+        }).fail(function () {
+            self.setState({loading: false, notFound: true});
+        });
+    },
+    render: function () {
+        var state = this.state;
+        if (state.notFound) {
+            return <div id="page-service">
+                <div className="page-header">
+                    <h2>Service Not Found</h2>
+                </div>
+                <p>The Service with ID <code>{this.props.uuid}</code> could not be found.</p>
+            </div>;
+        }
+
+        if (state.loading) {
+            return <div id="page-service">
+                <div className="loading">
+                    <i className="fa fa-circle-o-notch fa-spin" /> Fetching Service
+                </div>
+            </div>;
+        }
+        var service = state.service;
+        var metadata = service.metadata || {};
+        var params = service.params;
+        var customerMetadata = params.customer_metadata || {};
+        var tags = params.tags || {};
+
+        return <div id="page-service">
+            <div className="page-header">
+                <h1>
+                    <span className="service-name">{service.name}</span>&nbsp;
+                    <small className="uuid selectable">{service.uuid}</small>&nbsp;
+                    <span className="type">{service.type}</span>
+                </h1>
+            </div>
+            {this.state.error && <ErrorAlert error={this.state.error} />}
+            <section className="params">
```

Comment:
Angela Fong
8:55 AM

This entire section (L75:L114) is actually not required to be displayed for a service: 1) Customer metadata are not applicable to services, 2) Tags are view-only and aren't supposed to be something user has control over.